docs-kits/kits/Connector Kit/Adoption View/policies-in-catena.md

```diff
@@ -0,0 +1,230 @@
+| `purpose.trace.v1.qualityanalysis` | Notification API      | The data can be used for quality analysis to identify and select affected components and to send quality notifications to affected customers or suppliers |
+
+A `odrl:Constraint` referencing this purpose looks like this:
+
+```json
+{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "odrl:leftOperand": "https://w3id.org/catenax/policy/UsagePurpose",
+  "odrl:operator": {
+    "@id": "odrl:eq"
+  },
+  "odrl:rightOperand": "purpose.trace.v1.qualityanalysis"
+}
+```
+
+### Contract References
+
+Contract References by default aren't checked against credentials either. They are a vehicle to refer to contracts that
+are not governed by the Catena-X association - for instance bilaterally. Referencing such a contract's identifier can
+be achieved via an `odrl:Constraint` like this:
+
+```json
+{
+  "@context": {
+    "odrl": "http://www.w3.org/ns/odrl/2/"
+  },
+  "odrl:leftOperand": "https://w3id.org/catenax/policy/ContractReference",
+  "odrl:operator": {
+    "@id": "odrl:eq"
+  },
+  "odrl:rightOperand": "contract-123456789"
+}
+```
+
+### Chaining Constraints
+
+If a Policy is supposed to hold multiple constraints, Data Providers may chain them via a logical AND. This can be
+achieved via an `odrl:and` object encapsulating multiple other `odrl:Constraint`s or entering a list of them
+into the `odrl:constraint` property. The example below contains both versions.
+
+Constraints that are supposed to be checked with a logical OR should be published as separate Data Offers.
+
+## Example
+
+This specific Catalog contains one single `dcat:Dataset`, called "json-1-paper". It is the only entry in the top-level
+`dcat:dataset` property. To access this Dataset, the Consumer can choose between four Offers (see the `odrl:hasPolicy`
+property):
+
+- `"Y29udHJhY3QtYmlsYXRlcmFsLXBhcGVyLWV4YW1wbGUtMg==:anNvbi0xLXBhcGVy:ZDA4ZDM5OTgtOGY5ZS00MzBmLThjZDEtZmYwOWQxMmQxYzk5"`
+- `"Y29udHJhY3QtYmlsYXRlcmFsLXBhcGVyLWV4YW1wbGUtMQ==:anNvbi0xLXBhcGVy:ODFkMDI2MWYtNDNlNi00ZTIxLWJkMWYtZmFmZTI3MWQwYzhj"`
+
+```json
+{
+  "@context": {
+    "@vocab": "https://w3id.org/edc/v0.0.1/ns/",
+    "edc": "https://w3id.org/edc/v0.0.1/ns/",
+    "tx": "https://w3id.org/tractusx/v0.0.1/ns/",
+    "dcat": "http://www.w3.org/ns/dcat#",
+    "dct": "https://purl.org/dc/terms/",
+    "odrl": "http://www.w3.org/ns/odrl/2/",
+    "dspace": "https://w3id.org/dspace/v0.8/"
+  },
+  "@id": "693e9b66-04f2-4bfb-b3cd-daf5857b47c9",
+  "@type": "dcat:Catalog",
+  "dcat:dataset": [
+    {
+      "@id": "json-1-paper",
+      "@type": "dcat:Dataset",
+      "odrl:hasPolicy": [
+        {
+          "@id": "Y29udHJhY3QtYmlsYXRlcmFsLXBhcGVyLWV4YW1wbGUtMg==:anNvbi0xLXBhcGVy:ZDA4ZDM5OTgtOGY5ZS00MzBmLThjZDEtZmYwOWQxMmQxYzk5",
+          "@type": "odrl:Set",
+          "odrl:permission": {
+            "odrl:action": {
+              "odrl:type": "http://www.w3.org/ns/odrl/2/use"
```

Comment:
just a hint, this is actually a bug https://github.com/eclipse-tractusx/tractusx-edc/issues/1098